src/sagas.js

```diff
@@ -245,6 +284,12 @@ function* startVm (action) {
   yield stopProgress({ vmId: action.payload.vmId, name: 'start', result })
 }
 
+function* startPool (action) {
+  yield callExternalAction('startPool', Api.startPool, action)
+  yield put(getAllVms({ shallowFetch: true }))
```

Comment:
Please do not start refresh here, it might conflict with the scheduled polling.
If the 1 minute delay to see positive response becomes an issue, it will be resolved systematically.

Use the startProgress/stopProgress.